es6-classes/circle.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
+    super(area, circumference);
+    this.radius = radius;
+  }
+
+  print() {
+    console.log(`The radius of the Circle is ${this.radius}`);
```

Comment:
You need to call super.print() to get the Shape's properties.
This should return a string, not log directly.